prisma/migrations/20220709135708_add_category_table/migration.sql

```diff
@@ -0,0 +1,15 @@
+-- CreateTable
+CREATE TABLE "Category" (
+    "id" SERIAL NOT NULL,
+    "name" TEXT NOT NULL,
+    "decription" TEXT,
+    "coverImage" TEXT,
+    "createAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updateAt" TIMESTAMP(3) NOT NULL,
```

Comment:
Check this docs: https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#updatedat

Prisma has a special annotation to automatically update the updatedAt

Reply:
Also for createdAt